src/bot/constants/info/text.py

```diff
@@ -0,0 +1 @@
+MESSAGE = 'Здесь должно быть приветственное сообщение (Заглушка)!'
```

Comment:
Название константы должно быть информативным.
```suggestion
START_MESSAGE = 'Здесь должно быть приветственное сообщение (Заглушка)!'
```